tests/Unit/RollbackServiceV1Test.php

```diff
@@ -0,0 +1,240 @@
+<?php
```

Comment:
Not a big deal for these renamed files, but there is way to add them in git that would have made them show in the version control history as renamed and not newly added.

Reply:
How could I've done that?
In any case, the diff is shown for the v2 files, which is helpful.

Reply:
git mv OLD-FILENAME NEW-FILENAME